NumberBaseball/NumberBaseball/main.swift

```diff
@@ -4,7 +4,68 @@
 //  Copyright © yagom academy. All rights reserved.
 // 
 
-import Foundation
+var randomNumbers: [Int] = []
+var remainingChance: Int = 9
+let numbersCount: Int = 3
+let numberRange: ClosedRange<Int> = 1...9
 
-print("Hello, World!")
+func generateRandomNumbers(count: Int) -> [Int] {
+    var numbers: Set<Int> = []
+    
+    while numbers.count < count {
+        let randomNumber: Int = Int.random(in: numberRange)
+        
+        numbers.insert(randomNumber)
+    }
+    
+    return Array(numbers)
+}
 
+func checkTheResult(for inputNumbers: [Int]) -> (strikeCount: Int, ballCount: Int) {
+    var strikeCount: Int = 0
+    var ballCount: Int = 0
+    
+    for index in 0..<numbersCount{
+        if randomNumbers[index] == inputNumbers[index] {
+            strikeCount += 1
+        } else if randomNumbers.contains(inputNumbers[index]) {
+            ballCount += 1
+        }
+    }
```

Comment:
이전 코멘트에서는 아래 내용을 제안 드린 것이에요. 현행을 유지할지, 아래 내용을 적용할지 여부는 논의하셔서 결정하시면 됩니다!

반복문의 코드블럭 공백 형식도 신경써주세요~!

```suggestion
    for index in 0..<numbersCount {
        let isStrike: Bool = randomNumbers[index] == inputNumbers[index]
        let isBall: Bool = randomNumbers.contains(inputNumbers[index])
    
        if isStrike {
            strikeCount += 1
        } else if isBall {
            ballCount += 1
        }
    }
```